torchvision/models/vision_transformer.py

```diff
@@ -587,6 +605,8 @@ class ViT_H_14_Weights(WeightsEnum):
                     "acc@5": 97.730,
                 }
             },
+            "_flops": 167.295109,
+            "_weight_size": 2411.208604,
```

Comment:
It's going to be super hard to check that all these values were copied correctly. We should add tests on a follow up PR as discussed offline.